spec/spec_helper.rb

```diff
@@ -17,10 +19,12 @@
   end
 end
 
-def xmlpp(x)
-  s = ""
-  f = REXML::Formatters::Pretty.new(2)
-  f.compact = true
-  f.write(REXML::Document.new(x), s)
-  s
+def html_body_first_content_div(html)
+  Nokogiri::HTML(html).xpath('//div[@class="sect1"]')
 end
+
+def metanorma_process(input)
+  Metanorma::Input::Asciidoc
+    .new
+    .process(input, "test.adoc", :html)
```

Comment:
Can we use `:xml` for comparison? Because we want to compare the semantic XML, not the presentational HTML.

Reply:
@ronaldtse changed output format to docbook semantic xml